src/components/modules/ModuleCheckout.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  IOColors,
+  IOModuleIDPSavedVSpacing,
+  IOModuleStyles,
+  useIOExperimentalDesign
+} from "../../core";
+import { IOLogoPaymentType, LogoPayment } from "../logos";
+import { VSpacer } from "../spacer";
+import { Body, H6 } from "../typography";
+import { PressableModuleBase } from "./PressableModuleBase";
+
+type ModuleCheckoutProps = { ctaText: string } & (
+  | {
+      paymentLogo?: IOLogoPaymentType;
+      header: string;
+      body: string;
+      isLoading?: false;
+      onPress?: () => void;
+    }
+  | {
+      isLoading: true;
+    }
+);
+
+type CtaOnlyProps = { text?: string };
+
+export const ModuleCheckout = (props: ModuleCheckoutProps) => {
+  if (props.isLoading) {
+    return <LoadingVersion text={props.ctaText} />;
+  }
+  return (
+    <PressableModuleBase onPress={props.onPress}>
+      {props.paymentLogo && <LogoPayment name={props.paymentLogo} />}
+      <View>
+        <H6>{props.header}</H6>
+        <Body>{props.body}</Body>
+      </View>
+      <CTA text={props.ctaText} />
+    </PressableModuleBase>
+  );
+};
+
+const CTA = ({ text }: CtaOnlyProps) => {
+  const isExperimental = useIOExperimentalDesign();
+  const blue: IOColors = isExperimental ? "blueIO-500" : "blue";
+
+  return (
+    <Body color={blue} weight="SemiBold">
+      {text}
+    </Body>
+  );
+};
+
+const LoadingVersion = ({ text }: CtaOnlyProps) => (
+  <View
+    style={[
+      IOModuleStyles.button,
+      { paddingVertical: IOModuleIDPSavedVSpacing }
+    ]}
+  >
+    <Placeholder.Box animate="fade" radius={8} height={24} width={24} />
+    <View>
+      <Placeholder.Box animate="fade" radius={8} height={20} width={170} />
+      <VSpacer size={8} />
+      <Placeholder.Box animate="fade" radius={8} height={16} width={116} />
+    </View>
+    <CTA text={text} />
+  </View>
```

Comment:
In this component we need to include a new `View` container to avoid `space-between` behavior between the payment logo, text and related action. We don't need to use this extra `paddingVertical` style property as well:
```suggestion
<View style={IOModuleStyles.button}>
    <View style={[IOStyles.row, IOStyles.alignCenter]}>
      <Placeholder.Box animate="fade" radius={8} height={24} width={24} />
      <HSpacer size={8} />
      <View>
        <Placeholder.Box animate="fade" radius={8} height={20} width={170} />
        <VSpacer size={8} />
        <Placeholder.Box animate="fade" radius={8} height={16} width={116} />
      </View>
    </View>
    <CTA text={text} />
  </View>
```